target/linux/ath79/dts/qca9531_alfa-network_r36a.dtsi

```diff
@@ -87,16 +87,21 @@
 				reg = <0x070000 0x010000>;
 				read-only;
 
-				compatible = "nvmem-cells";
-				#address-cells = <1>;
-				#size-cells = <1>;
 
```

Comment:
It seems that there are some unnecessary new lines. '\n\n\n' can be replaced with '\n\n'.

Reply:
Fixed.